### ISSUE TITLE
License

### ISSUE THREAD

**Issue by chpatrick** [adasworks/ocamcalib-plus#6]
Hi (Sziasztok! :D)

We'd like to use some parts of this project. Would it be possible to license it under an MIT or BSD-style license?

Thanks!

**Comment by tamaskenez** [comment]
(keeping it in English for those readers)

Actually my boss asked me to take this down a few weeks ago but I forgot it. Anyway I'm asking him.
Of course most code in this repo is the work of Scaramuzza. I assume you're interested in the C++ port.

**Comment by chpatrick** [comment]
Yes, that's right. We're only using a few lines and we could always port the Scaramuzza code ourselves, but I thought I would ask first. :)

**Comment by tamaskenez** [comment]
Okay, I'm sorry, I need to take this repo down in a few days and move it to our internal server.
